Added FAQ entry on using Linux via remote desktop solution for windows users.

kunalghosh opened the pull request.

avehtari commented on FAQ.md:
```diff
@@ -31,3 +31,18 @@ install.packages("tinytex")
+## Windows user and have a lot of problems getting packages to install ?
+
+> ⚠️ Note that **Windows is not officially supported in the course** but we will try our best to help you get your setup (in Windows) working. This is because of two main reasons:
+> 1. `Stan`, the probabilistic programming language which we will later on use in the course, doesn't have good support for Windows. 
```
It should be said that "`Stan`, the probabilistic programming language requires C++ compiler toolchain which is not available by default in Windows (blame Microsoft)."

kunalghosh replied:
Just updated the FAQ. The new version should cover most of the points we discussed here.

kunalghosh commented:
Updated the FAQ with instructions directing to `RStan` setup page which has windows installation instructions. Also added a note about the C++ compiler tool-chain.

MansMeg commented:
Alright. Maybe do a PR. @avehtari ? You have any further suggestions?
